### DISCUSSION TITLE
RegexManager rules + nodeJS defaults

### DISCUSSION THREAD

**Discussion by setchy** [renovatebot/renovate#21191] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

`node` images follow the custom Regex Manager rule.

### What you tried so far.

_This is a follow-on from https://github.com/renovatebot/renovate/discussions/21094_
_Note: indentation issues are being discussed separately https://github.com/renovatebot/renovate/discussions/21169_

As demonstrated in https://github.com/setchy/renovate-bitbucket-pipelines, the regexManager rule that updates bitbucket-pipeline image statements to authenticated image objects is mostly working, with the exception of `node` images

* Regex rule working for sonarqube: https://github.com/setchy/renovate-bitbucket-pipelines/pull/7
* Regex rule being skipped for nodejs: https://github.com/setchy/renovate-bitbucket-pipelines/pull/6

🤷 

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by HonkingGoose** [comment]
Pinging @rarkins to take a look at this and decide what to do. 😉

**Reply by rarkins** [reply]
🤷 Tried it without any presets whatsoever? (e.g. drop config:base)

**Reply by setchy** [reply]
An update - node regex updates are now (mostly) working after following the advice from this related discussion item - https://github.com/renovatebot/renovate/discussions/21392#discussioncomment-5562658

there are still some scenarios whereby the PR isn't raised.  I don't fully know why thought. Debugging hasn't uncovered anything that stands out to me right away
